text/0000-general-fund.md

```diff
@@ -0,0 +1,101 @@
+
+- Title: general-fund-spending-guidelines
+- Authors: [Yeastplume](mailto:yeastplume@protonmail.org)
+- Start date: [Feb 07, 2020]
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)·
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+This RFC establishes a set of guidelines for spending the Grin Development Fund, the procedure via which funding requests can be made, and a set of procedures for managing and reporting on the fund.
+
+# Motivation
+[motivation]: #motivation
+
+Previously there were no clear guidelines on how the Grin Development Fund could be spent and what types of activities could and could not be financed from the fund. This RFC presents a set of non-exhaustive guidelines intended to ensure that:
+
+* Donators have a clear understanding of how their donations are likely to be spent.
+* Those with funding ideas have a clear reference point to understand what types of proposals are likely to be considered for funding.
+* Those with funding ideas have a specific process through which they can apply for funding.
+* The team making spending decisions has clear, community-established guidelines against which to consider new funding requests.
+* The decision-making process through which funding requests are granted is as fair, consistent and transparent as it can reasonably be.
+* The wider community understands how the fund is managed and that transparency is evident.
+
+Note that this document does not establish hard-and-fast rules for how the fund's contents can be spent. It is meant to be an indicative focal point for the community, and is intended to evolve with the needs of the project.
+
+# Community-level explanation
+[community-level-explanation]: #community-level-explanation
+
+## Appropriate and inappropriate uses of the Grin Development Fund
+
+Although spending decisions will always be made on a case-by-case basis, it is understood that the Grin Development Fund is to be used mainly to finance the development needs of the Grin project and the advancement of Grin/Mimblewimble as a whole. 
+
+There is scope for interpretation as to what this actually means, but the following presents (non-exhaustive) guidelines as to the types of activities that would be considered appropriate to finance from the fund.
+
+* Activities related to the continuing development needs of the Grin code-base and related projects under the GitHub [mimblewimble](github.com/mimblewimble) organization. This can include:
```

Comment:
One follow-up question rising from this point is how projects are admitted in the mimblewimble org. It may not be necessary to explain my question as part of this RFC but I think it is worth dicussing/documenting elsewhere at least.

Reply:
Good point @kargakis, you're also right that it's probably not suitable for this RFC. I created an issue and will look to add to next governance agenda: https://github.com/mimblewimble/grin-pm/issues/253